Makefile

```diff
@@ -0,0 +1,20 @@
+DEFAULT_GOAL := help
```

Comment:
please update the readme on the usage of this setup.

Reply:
Not very clear on this, can you be more specific?
This setup is for testing; Do you mean I should list in the Readme for them to run the the tests using this script?

Reply:
yes. like 'how to run local test' section.

Reply:
Done